README.md

```diff
@@ -48,18 +51,23 @@ To **resume uploading** a dataset whose upload was interrupted:
 
 `cleanlab dataset upload -f [dataset filepath] --id [dataset ID]`
 
-A dataset ID is generated and printed to the terminal the first time the dataset is uploaded. It can also be accessed by
-visiting https://app.cleanlab.ai/datasets and selecting 'Resume' for the relevant dataset.
+A dataset ID is generated and printed to the terminal the first time the dataset is uploaded. It can also be accessed from the Datasets section of the Cleanlab Studio dashboard by visiting https://app.cleanlab.ai/ and selecting 'Resume' for the relevant dataset.
 
 #### Generate dataset schema
 
 To generate a dataset schema (prior to uploading your dataset):
 
 `cleanlab dataset schema generate -f [dataset filepath]`
 
+* For `Modality (text, tabular, image): ` enter one of the following:
+  * `text` - to only find label errors based on a single column of text in your dataset.
+  * `tabular` - to find data and label issues based on any subset of the column features.
+  * `image` - to find label errors in your image dataset.
+
 * For `Id column: `, please enter the string name of of the column in your dataset that contains the id of each row.
-* For `Modality (text, tabular): `, please enter `text` to only find label errors based on a single column of text,
-  otherwise enter `tabular` to find data and label issues based on any subset of the column features.
+
+* For `filepath column: ` (only for image modality), please enter the string name of of the column in your dataset that contains the path to each image.
```

Comment:
typo: "of" is duplicated

Reply:
I'd clarify that the image files are allowed to live in a different folder as long as their paths are correctly specified in this column, or where exactly the image files are allowed to live if not anywhere.